examples/nvcontrol.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python
+#
+# examples/nvcontrol.py -- demonstrate the NV-CONTROL extension
+#
+#    Copyright (C) 2019 Roberto Leinardi <roberto@leinardi.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public License
+# as published by the Free Software Foundation; either version 2.1
+# of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the
+#    Free Software Foundation, Inc.,
+#    59 Temple Place,
+#    Suite 330,
+#    Boston, MA 02111-1307 USA
+
+
+# Python 2/3 compatibility.
+from __future__ import print_function
+
+import sys
+import os
+
+# Change path so we find Xlib
+from pprint import pprint
+
+from Xlib.display import Display
+from Xlib.ext.nvcontrol import Gpu, Cooler
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+
+if __name__ == '__main__':
+    display = Display()
+    # Check for extension
+    if not display.has_extension('NV-CONTROL'):
+        sys.stderr.write('%s: server does not have the NV-CONTROL extension\n'
+                         % sys.argv[0])
+        print(display.query_extension('NV-CONTROL'))
+        sys.stderr.write("\n".join(display.list_extensions()))
+        if display.query_extension('NV-CONTROL') is None:
```

Comment:
Not critical, but is there any need to call `query_extension` twice? Maybe save returned value to a variable and then print it and use it in a condition.

Reply:
Frankly, I just copy pasted this entire block from the Xinerama example without even thinking too much of it:  https://github.com/python-xlib/python-xlib/blob/master/examples/xinerama.py#L49

Let me know if I should change it or leave it like it is.

Reply:
I think this code can be changed. I'd prefer keeping technical debt not growing. `xinerama.py` is old and not necessary to change it right now (it's up to you).

Reply:
Sure no problem, I'll change that too :+1: